### ISSUE TITLE
Printing is borked

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#54]
```
Get a "permission-denied" error when printing.


```

Original issue reported on code.google.com by `dean.edw...@gmail.com` on 10 Mar 2008 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
I also get the javascript error when using the IE8.js script in IE6, and then
choosing either Print Preview or Print. 

The javascript error I'm getting is (as reported by Companion.JS, along with 
call stack)
Permission denied (line 1089)
* line 1125
* line 1132
* line 1032
* line 215
* line 514

It looks as though IE7 is throwing an error as well. 


It looks like this is also reported in #94:
http://code.google.com/p/ie7-js/issues/detail?id=94&q=print

and #155
http://code.google.com/p/ie7-js/issues/detail?id=155&q=print

```

Original comment by `leombr...@gmail.com` on 9 Sep 2009 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
I've had this issue with IE6 & IE7 using IE8.js. I've modified the following 
line in 
the code and the error went away.

Original Code:
if(c=="all"||c==R.media)

Changed to:
if((c=="all"&&R.media!="print")||(c==R.media && R.media!="print"))

From what I've tested so far, the outcome is as expected.

Hope it helps.
Fabien D.
```

Original comment by `fabien.d...@gmail.com` on 3 Nov 2009 at 1:24

**Comment by GoogleCodeExporter** [comment]
```
Issue 94 has been merged into this issue.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:02

**Comment by GoogleCodeExporter** [comment]
```
Issue 155 has been merged into this issue.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:02

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 12 Feb 2010 at 2:03
- Changed state: **Fixed**
